protocol/contracts/admin/TempleTeamPaymentsFactory.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/proxy/Clones.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "./TempleTeamPaymentsV2.sol";
+contract TempleTeamPaymentsFactory is Ownable {
+    error AllocationsLengthMismatch();
+    error AllocationAddressZero();
+    error ClaimZeroValue();
+    error ImplementationAddressZero();
+
+    struct FundingData {
+        address paymentContract;
+        uint16 epoch;
+        uint256 totalFunding;
+    }
+
+    IERC20 public temple = IERC20(0x470EBf5f030Ed85Fc1ed4C2d36B9DD02e77CF1b7);
```

Comment:
make immutable

Reply:
might want to initialize this in the constructor so we could change it on a testnet